backend/btrixcloud/crawlconfigs.py

```diff
@@ -602,7 +598,15 @@ async def update_crawl_config(
             and ((not update.profileid) != (not orig_crawl_config.profileid))
         )
 
-        changed = changed or (orig_crawl_config.proxyId != update.proxyId)
+        # either unsetting profile or no profile set on current config
+        no_profile = update.profileid == "" or not orig_crawl_config.profileid
```

Comment:
I think if the update is adding a profile when there wasn't one before, the `not origi_crawl_config.profileid` will still be true, and so this will falsely return that there's no profile when we are in fact setting one.